sdk/eventhubs/messaging-eventhubs/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Release History
```

Comment:
What's motivating the folder rename, and removing `azure-` from `azure-messaging-eventhubs`? Other libraries, like storage, have names like `azure-storage-blobs`.

Reply:
Because the pipeline name created by the create-pipelines script was azure-messaging-eventhubs